### PR TITLE
release: the initial release of PoS

### PR DESCRIPTION
**Pull request by Shabirmean**
#### Description
- Release PoS MVP1

### CODE REVIEW

**Comment by Shabirmean** [comment]
:zap:  Release Status :zap: </br>
- **Version updated**: Version in the  has been updated to the next **patch** release version.
- **Publish Artifacts:** Check the most recent commit (45f524e99d42b0b7d8173cd55a4474482e51286c) for correctness and run the ["pos-publish-release-artifacts"](https://console.cloud.google.com/cloud-build/triggers;region=global?project=point-of-sale-ci) trigger against this branch **[release-patch]** to upload the artifacts _(Jars and Images)_ to Artifact registry.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1 +1,7 @@
 # Changelog
+
+All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
+
+### 0.1.0 (2022-05-12)
+
+🍔 This is the inital release of the Point of Sale Application. Please try the [Quickstart](/docs/quickstart.md) to deploy this application to your clusters.
```
Comment (NimJay) [inline CHANGELOG.md:7]: Praise: Good emoji.

**Review by ckim328 — DISMISSED** [review]

**Comment by Shabirmean** [comment]
:zap:  Two deployments have been created for the Point-of-Sale application. You may access and test them at: :zap: </br>
- **MySQL DB** backed deployment: [**34.70.155.178**](http://34.70.155.178)
- **Embedded H2** DB backed deployment: [**35.225.95.146**](http://35.225.95.146)

**Review by charlieyu1996 — DISMISSED** [review]
LGTM!

**Comment by Shabirmean** [comment]
:zap:  Two deployments have been created for the Point-of-Sale application. You may access and test them at: :zap: </br>
- **MySQL DB** backed deployment: [**34.70.155.178**](http://34.70.155.178)
- **Embedded H2** DB backed deployment: [**35.225.95.146**](http://35.225.95.146)

**Review by NimJay — APPROVED** [review]
LGTM!

**Comment by Shabirmean** [comment]
:zap:  Two deployments have been created for the Point-of-Sale application. You may access and test them at: :zap: </br>
- **MySQL DB** backed deployment: [**34.70.155.178**](http://34.70.155.178)
- **Embedded H2** DB backed deployment: [**35.225.95.146**](http://35.225.95.146)

**Comment by Shabirmean** [comment]
:zap:  Release Status :zap: </br>
- **Artifacts published**: [Maven JARs and Container images](https://console.cloud.google.com/artifacts/browse/point-of-sale-ci?project=point-of-sale-ci) for release version **v0.1.0** have been published to Artifact Registry
- **Manifests updated**: [Release manifests](https://github.com/GoogleCloudPlatform/point-of-sale/tree/release-patch/k8-manifests/release) have been updated to use the version **v0.1.0** images
- **Version bumped:** Version in the _pom.xml/package.json_ has been bumped (94d350277a19b9f3e0d2e26dc6149d1a4a1e8027) to the snapshot version of the next **patch** release **v0.1.1-SNAPSHOT**
- **Github release created:** A draft [Github release](https://github.com/GoogleCloudPlatform/point-of-sale/releases) has been created. Inspect and **publish it**
- **Merge PR:** Inspect the latest changes and merge the PR to **main** to complete the release.

:rocket: Don't forget to run the ["pos-deploy-release"](https://console.cloud.google.com/cloud-build/triggers;region=global?project=point-of-sale-ci) trigger against the **main** to deploy the latest release!!!

**Pull request merged**